OpenRA.Mods.Common/Widgets/Logic/Ingame/GameTimerLogic.cs

```diff
@@ -44,7 +44,7 @@ public GameTimerLogic(Widget widget, OrderManager orderManager, World world)
 			{
 				timer.GetText = () =>
 				{
-					if (status == null && shouldShowStatus())
+					if (status == null && world.ReplayTimestep == 0 && shouldShowStatus())
```

Comment:
Does this create dead code above?

Reply:
There shouldn't be any dead code

Reply:
Ah, I see what you mean. The `statusText` is still being used when the UI defines `GAME_TIMER_STATUS`